cisco_umbrella_dns/manifest.json

```
@@ -0,0 +1,53 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "9f98de10-9c98-4601-ae36-cbe25c4be018",
+    "app_id": "cisco-umbrella-dns",
+    "display_on_public_website": false,
```

```suggestion
    "display_on_public_website": true,
```
@romandatadog should this be true?

---

no we don't want to make it go live immediately